packages/preview/strath-maestrum/0.0.1/src/template.typ

```diff
@@ -0,0 +1,61 @@
+#import "@preview/wordometer:0.1.5": *
+
+#let report(
+  class: [],
+  title: [],
+  author: [],
+  number: [],
+  supervisor: [],
+  date: [],
+  abstract: [],
+  coverpage-image-path: none, header-image-path: none,
```

Comment:
[Taking a path here will not work](https://github.com/typst/packages/blob/main/docs/resources.md#paths-are-package-scoped). You should take `content` (i.e a call to `image(...)`) directly.